1-module/1-module.js

```diff
@@ -1,31 +1,31 @@
 const btn = document.querySelector(".start-game");
 
-function month() {
-  const n = prompt("Введите число от 1-12");
+function getMonthByNumber() {
+  const monthNumber = prompt("Введите число от 1-12");
 
-  if (n == 1) {
+  if (monthNumber == 1) {
```

Comment:
Действительно, данная функция стала выглядеть лучше. Но мне кажется, что ее можно в разы упростить.

```suggestion
const months = ["Январь"];

const month = months[monthNumber];
```

Reply:
Согласен! Но данные изменения выходят за рамки данной задачи/ветки. Я сделаю это отдельно